app/src/main/res/layout/loyalty_card_view_layout.xml

```diff
@@ -152,10 +153,12 @@
         <ImageView
             android:importantForAccessibility="no"
             android:id="@+id/fullscreen_image"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="0dp"
             app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toTopOf="@+id/scaler_guideline"/>
+            app:layout_constraintBottom_toTopOf="@+id/scaler_guideline"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toLeftOf="@+id/scaler_widthguideline"/>
```

Comment:
`constraintLeft` and `constraintRight` are rarely the correct choices, as they force things to the left/right side of the screen, even in cultures which use RTL instead of LTR. `constrainedStart` and `constrainedEnd` would make more sense here I think, it would use the left of the screen as the "start" in LTR cultures, while using the right of the screen as the start in RTL cultures, which I would guess is more like what people would consider "normal" (human culture is complex :))

I'm also wondering: is there any specific reason you've chosen to force it to a side of the screen instead of keeping it centered?

(Sidenote, the white background not staying in the whole area looks odd, but that was kind of already an issue before but became much more noticeable)